src-electron/db/query-zcl.js

```diff
@@ -449,8 +452,7 @@ async function selectAllStructItemsById(db, id) {
 SELECT
   STRUCT_ITEM.FIELD_IDENTIFIER,
   STRUCT_ITEM.NAME,
-  (SELECT DATA_TYPE.NAME FROM DATA_TYPE WHERE DATA_TYPE.DATA_TYPE_ID = STRUCT_ITEM.DATA_TYPE_REF) AS TYPE,
-  DATA_TYPE.NAME AS DATA_TYPE_REF_NAME,
```

Comment:
This could break the templates for backwards compatibility.

Reply:
No templates use these values (& rightfully so, they're junk values, see https://github.com/project-chip/zap/pull/1501#discussion_r1902229928)

Reply:
My general experience is that lots of good deeds result in moving the needle in a positive direction

Yes, sometimes they lead to bugs in unforeseen cases

But the alternative, letting debt build up because it's scary to clean things up, leads to more issues over time